src/components/TimeWithValuesGraph/TimeWithValuesGraph.css

```diff
@@ -0,0 +1,7 @@
+.bg-color {
```

Comment:
I suggest naming the class as the element it gives style too. bg-color is too abstract IMHO